src/Connection.js

```diff
@@ -27,7 +28,21 @@ export class Connection {
     this.wsInBuffer = new Buffer(0);
     this.handshakeComplete = false;
     this.isClosed = false;
-    this.dbSocket = net.createConnection(dbPort, dbHost);
+    let options = {
+      host: dbHost,
+      port: dbPort
+    };
+    if (dbSsl === null) {
+      this.dbSocket = net.connect(options);
```

Comment:
Can you make this part become the `else` clause instead of explicitly checking if it's `null`, so users that pass in unsupported values like `undefined` or `false` end up connecting without tls?